plugins/hypervisors/vmware/src/main/java/org/apache/cloudstack/storage/motion/VmwareStorageMotionStrategy.java

```diff
@@ -210,7 +173,7 @@ private VirtualMachine getVolumeVm(DataObject srcData) {
     private Pair<Long, String> getHostIdForVmAndHostGuidInTargetClusterForAttachedVm(VirtualMachine vm,
                                                                                      StoragePool targetPool,
                                                                                      ScopeType targetScopeType) {
-        Pair<Long, Long> clusterAndHostId = findClusterAndHostIdForVm(vm);
+        Pair<Long, Long> clusterAndHostId = vmManager.findClusterAndHostIdForVm(vm);
```

Comment:
With the above comment, this should be passing vm.getId() instead

Reply:
Done